src/body-renderer.js

```diff
@@ -82,12 +83,25 @@ export default class BodyRenderer {
                 column: col
             };
         });
+
+        const rowCount = this.visibleRows.length;
         const totalRow = this.visibleRows.reduce((acc, prevRow) => {
             return acc.map((cell, i) => {
                 const prevCell = prevRow[i];
-                if (typeof prevCell.content === 'number') {
-                    cell.content += prevRow[i].content;
+
+                let useDefaultAccumulator = true;
+                if (this.events.accumulator) {
```

Comment:
`events` doesn't feel the right place to put this. Put this in `option.hooks`. So this will be called like `this.options.hooks.totalAccumulator`

Reply:
Done